packages/clay-css/src/scss/mixins/_forms.scss

```diff
@@ -701,282 +701,317 @@
 /// - Add @link to documentation
 
 @mixin clay-select-variant($map) {
-	$enabled: setter(map-get($map, enabled), true);
-
-	$base: setter($map, ());
-	$base: map-merge(
-		$base,
-		(
-			background-color:
-				setter(map-get($map, bg), map-get($map, background-color)),
-			background-clip:
-				setter(map-get($map, bg-clip), map-get($map, background-clip)),
-			background-image:
-				setter(map-get($map, bg-image), map-get($map, background-image)),
-			background-position:
-				setter(
-					map-get($map, bg-position),
-					map-get($map, background-position)
-				),
-			background-repeat:
-				setter(
-					map-get($map, bg-repeat),
-					map-get($map, background-repeat)
-				),
-			background-size:
-				setter(map-get($map, bg-size), map-get($map, background-size)),
-			color:
-				setter(
-					map-get($map, color),
-					if(
-						variable-exists(input-color),
-						$input-color,
+	@if (type-of($map) == 'map') {
```

Comment:
Hey @pat270 can we do an early return to not change the indentation?

```sass
@if (type-of($map) != 'map') {
  @return
}
```

Reply:
@matuzalemsteles good idea, unfortunately mixins don't return values. I can't think of another way around it atm.

Reply:
Oh yes that's true, that's ok, let's go with it!